pybsn/__init__.py

```diff
@@ -467,6 +468,16 @@ def __repr__(self):
         return "BigDbClient(%s)" % self.url
 
 
+def _normalize(self, v):
```

Comment:
If I get this correct for the queries, the queries for each of the cases need to look like this:

If the code specifies boolean True the query looks like:

/container/list[key=true]

But string "true"

/container/list[key="true"]

Similar for ints int  123 and string "123"

/container/list[key=123]

and

/container/list[key="123"]

Floodlight will allow all four above and convert correctly depending the schema.

Stings will always be:

/container/list[key="string"]

Is that right ?

Reply:
Yes that's correct.

From the pybsn side this normalize function will have input -> output:
True -> true
"true" -> "true"
False -> false
"foo" -> "foo"
123 -> 123

Apart from the booleans, the overall behaviour here should remain unchanged

Floodlight will handle converting things like "true" to true, or "123" to 123.

Reply:
Thanks Luke seems good.